src/map/mob.hpp

```diff
@@ -28,6 +28,8 @@ struct guardian_data;
 #define MAX_MVP_DROP_ADD 2
 #define MAX_MOB_DROP_TOTAL (MAX_MOB_DROP+MAX_MOB_DROP_ADD)
 #define MAX_MVP_DROP_TOTAL (MAX_MVP_DROP+MAX_MVP_DROP_ADD)
+//Dynamic item drop ratio database for per-item drop ratio modifiers overriding global drop ratios.
+#define MAX_ITEMRATIO_MOBS 10
```

Comment:
```suggestion
```
Do we truly want to limit this? I say dump it. Free ourselves from these limits of static arrays!

Just remember to include this define in csv2yaml.hpp for the converter.